pkgs/tools/security/nitrokey-app2/pyqt5-stubs.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pyqt5
+, pyqtwebengine
+, pyqtchart
+, pyqt3d
+, pyqtdatavisualization
+, pytestCheckHook
+, mypy
+, pytest-xvfb
+}:
+
+buildPythonPackage rec {
+  pname = "pyqt5-stubs";
+  version = "5.15.6.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "python-qt-tools";
+    repo = "PyQt5-stubs";
+    rev = version;
+    hash = "sha256-qWnvlHnFRy8wbZJ28C0pYqAxod623Epe5z5FZufheDc=";
+  };
+
+  propagatedBuildInputs = [
+    pyqt5
+    pyqtwebengine
+    pyqtchart
+    pyqt3d
+    pyqtdatavisualization
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    mypy
```

Comment:
```suggestion
```
we don't want to run linters

Reply:
I had to `postPatch`-away an entire test file to remove the dependency to mypy, but it works now without it.